lib/application/widgets/theme/custom_text_field.dart

```diff
@@ -114,7 +118,7 @@ class CustomTextField extends HookConsumerWidget {
 
     final textField = Container(
       decoration: BoxDecoration(
-        border: _hasErrorText ? Border.all(color: theme.destructiveSwatch, width: dimens.genericBorderHeight) : null,
+        color: _hasErrorText ? theme.destructiveSwatch : null,
```

Comment:
Had some animation issues when using `border`. Using just `color` with `borderRadius` solved the problem (not sure why, but I gave up while trying to understand)

Reply:
If they achieve the same layout that's fine, but does it though?

Reply:
They have the exactly same visual behavior, but `border` has some issues when animating. I can't give you more details because I lost a lot of time trying to debug and understand, but couldn't figure it out